modules/manage/partials/tiered-storage.adoc

```diff
@@ -1510,10 +1457,116 @@ helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --crea
+CAUTION: Do not set an object storage property to an empty string `""` or to `null` as a way to reset it to its default value.
 
 endif::[]
 
+[[max-cache]]
+=== Set a maximum cache size
+
+To ensure that the cache does not grow uncontrollably, which could lead to performance issues or disk space exhaustion, you can control the maximum size of the cache.
+
+Redpanda checks the cache periodically according to the interval set in config_ref:cloud_storage_cache_check_interval,true,properties/object-storage-properties[]. If the size of the stored data exceeds the configured limit, the eviction process starts. This process removes segments that haven't been accessed recently until the cache size drops to the target level.
+
+*Related properties*:
+
+- config_ref:cloud_storage_cache_size_percent,true,properties/object-storage-properties[]
+- config_ref:cloud_storage_cache_max_objects,true,properties/object-storage-properties[]
+- config_ref:cloud_storage_cache_size,true,properties/object-storage-properties[]
+
+*Recommendation*: By default, `cloud_storage_cache_size_percent` is tuned for a shared disk configuration. If you are using a dedicated cache disk, consider increasing this value.
+
+==== Cache trimming
+
+Cache trimming helps to balance optimal cache use with the need to avoid blocking reads due to a full cache. The trimming process is triggered when the cache exceeds certain thresholds relative to the <<max-cache, maximum cache size>>.
+
+*Related properties*:
+
+- config_ref:cloud_storage_cache_trim_threshold_percent_objects,true,properties/object-storage-properties[]
+- config_ref:cloud_storage_cache_trim_threshold_percent_size,true,properties/object-storage-properties[]
+- config_ref:cloud_storage_cache_trim_carryover_bytes,true,properties/object-storage-properties[]
+- config_ref:cloud_storage_cache_trim_walk_concurrency,true,properties/object-storage-properties[]
+
+*Recommendations*:
+
+- A threshold of 70% is generally recommended for most use cases. This helps balance optimal cache use and ensuring there is enough free space to handle sudden spikes in data without blocking reads. For example, setting `cloud_storage_cache_trim_threshold_percent_size` to 80% means that the cache trimming process will start when the cache takes up 80% of the maximum cache size.
+
+- Monitor the behavior of your cache and the performance of your Redpanda cluster. If you notice that reads are taking unexpectedly long or you encounter timeout errors, it might be a sign that your cache is filling up too quickly. In these cases, consider lowering the thresholds to trigger trimming earlier.
+
+NOTE: The lower you set the threshold, the earlier the trimming will start, but it also means wasting cache space. Setting a higher threshold uses cache space more efficiently but risks blocking reads if the cache fills up too quickly. Adjust the settings based on your workload and monitor the cache performance to find the right balance for your environment.
```

Comment:
```suggestion
NOTE: The lower the threshold, the earlier trimming starts, but it can also waste cache space. Setting a higher threshold uses cache space more efficiently, but it risks blocking reads if the cache fills up too quickly. Adjust the settings based on your workload, and monitor the cache performance to find the right balance for your environment.
```